odoo/addons/base/tests/test_mimetypes.py

```diff
@@ -26,6 +26,16 @@
 ZIP = b"""UEsDBBQACAAIAGFva1AAAAAAAAAAAAAAAAAFACAAdC50eHRVVA0AB5bgaF6W4GheluBoXnV4CwABBOgDAAAE6AMAAA
 MAUEsHCAAAAAACAAAAAAAAAFBLAQIUAxQACAAIAGFva1AAAAAAAgAAAAAAAAAFACAAAAAAAAAAAACkgQAAAAB0LnR4dFVUDQAHlu
 BoXpbgaF6W4GhedXgLAAEE6AMAAAToAwAAUEsFBgAAAAABAAEAUwAAAFUAAAAAAA=="""
+# Sample XML file: an empty SEPA Direct Debit file
+XML = b"""PD94bWwgdmVyc2lvbj0nMS4wJyBlbmNvZGluZz0ndXRmLTgnPz4KPERvY3VtZW50IHhtbG5zOnhzaT0iaHR0cDovL
+3d3dy53My5vcmcvMjAwMS9YTUxTY2hlbWEtaW5zdGFuY2UiIHhtbG5zPSJ1cm46aXNvOnN0ZDppc286MjAwMjI6dGVjaDp4c2Q6
+cGFpbi4wMDEuMDAxLjAzIj4KICA8Q3N0bXJDZHRUcmZJbml0bj4KICAgIDxHcnBIZHI+CiAgICAgIDxNc2dJZD4xMjM0NTZPZG9
+vIFMuQS44OTM4NzM3MzM8L01zZ0lkPgogICAgICA8Q3JlRHRUbT4yMDE4LTExLTIxVDA5OjQ3OjMyPC9DcmVEdFRtPgogICAgIC
+A8TmJPZlR4cz4wPC9OYk9mVHhzPgogICAgICA8Q3RybFN1bT4wLjA8L0N0cmxTdW0+CiAgICAgIDxJbml0Z1B0eT4KICAgICAgI
+CA8Tm0+T2RvbyBTLkEuPC9ObT4KICAgICAgICA8SWQ+CiAgICAgICAgICA8T3JnSWQ+CiAgICAgICAgICAgIDxPdGhyPgogICAg
+ICAgICAgICAgIDxJZD5CRTA0Nzc0NzI3MDE8L0lkPgogICAgICAgICAgICAgIDxJc3NyPktCTy1CQ0U8L0lzc3I+CiAgICAgICA
+gICAgIDwvT3Rocj4KICAgICAgICAgIDwvT3JnSWQ+CiAgICAgICAgPC9JZD4KICAgICAgPC9Jbml0Z1B0eT4KICAgIDwvR3JwSG
+RyPgogIDwvQ3N0bXJDZHRUcmZJbml0bj4KPC9Eb2N1bWVudD4K"""
```

Comment:
It seems unecessary to store xml (which is a text format) in base64. Base64 is used for the other files because they use a binary format

Reply:
Thanks for the review, I made the changes.

> How are svg images mimetyped now?

With this PR, SVG images still have the correct mime type `image/svg+xml`

Reply:
Indeed, I didn't see the test at L92